manuscript/chapter2-philosophy.md

```diff
@@ -114,7 +116,17 @@ Another point is in a OSS project is not possible to do questions it isn't OSS.
 
 ## How to evolve it to the next Pokemon
 
-To open some tickets (or participate into existing ones) is very important, it allows you to get involved at a very low level and to join the party.
+To open some tickets (or participate into existing ones) is very important, it allows you to get involved at a very low level and to join the party.  
+For a lot of people the real question is: how I can live with that? The famous "pagnotta" already mentioned. Well, RedHat is the most profit company about Open Source now part of the IBM released a [report of 2019-4](https://www.linuxfoundation.org/resources/open-source-guides/winding-down-an-open-source-project/) about the enterprise world with OSS.  
+The main points:
+
+* Modernized the companies
+* Improved the security of their infrastructure
+* Lower total costs
+* Integration with others platform they use
+* Ability to customize the software
+
+Those are just few hints from this reports that show how can be powerful embrace this philosophy and solutions. For IT companies also open new doors for business like this [Tidelift's report explains to getting more developers](https://tidelift.com/subscription/managed-open-source-survey?hsCtaTracking=230a6d8b-3c6c-4c5e-9583-efba427f66e8%7Cd15d90a8-d5c2-47b4-83fe-81602f6f126c).
```

Comment:
Can you safely remove the Call-to-action tracking code?

Frank @karlitschek s talks about running an OSS company / why GPL is good for business is getting traction this year as well.
Perhaps include it in your book?

https://fosdem.org/2020/schedule/event/gpl_and_business/
https://changelog.com/podcast/383
https://foss-north.se/2020/speakers-and-talks.html#fkarlitschek
to name a few

Reply:
I already mentioned his work in the book :-)